reverse_replication/launcher.go

```diff
@@ -45,6 +45,10 @@ var (
 	sourceShardsFilePath string
 	sessionFilePath      string
 	machineType          string
+	vpcNetwork           string
+	vpcSubnetwork        string
+	vpcProjectId         string
```

Comment:
Nit: Suggest renaming to `vpcHostProjectId` to keep parity with GCP docs - 
https://cloud.google.com/dataflow/docs/guides/specifying-networks#shared:~:text=that%20HOST_PROJECT_ID